src/sidebar/tabcenter.css

```diff
@@ -826,31 +846,3 @@ body.dark-theme {
 body.dark-theme #newtab-menu {
     color: black;
 }
-
-body.dark-theme #searchbox-icon {
```

Comment:
The dark theme looked fine to me after testing your patch, but please double check this didn't break anything.